lib/sorted_set.js

```diff
@@ -224,7 +224,9 @@ lunr.SortedSet.prototype.union = function (otherSet) {
 
   unionSet = longSet.clone()
 
-  unionSet.add.apply(unionSet, shortSet.toArray())
+  for(var i = 0, shortSetElements = shortSet.toArray(); i < shortSetElements.length; i++){
+    unionSet.add(shortSetElements[i]);
```

Comment:
tiny nitpick - none of the other code includes explicitly semicolons. We should probably try and match that style.